documentation/CONFIG.md

```diff
@@ -251,7 +250,7 @@ Also, these topic level configurations will be override the behavior of Connecto
 ``scylladb.offset.storage.table.enable``
 
   If true, Kafka consumer offsets will be stored in ScyllaDB table. If false, connector will skip writing offset 
```

Comment:
will `also` be stored in ScyllaDB table

precising it's in plus and do not replace the offsets managed by connect framework ?

Reply:
@avelanarius do you agree ?

Reply:
Sorry for the late reply. In Scylla offset storage mode, the connector does not save any offsets to Kafka (only Scylla).

Reply:
ok so the most explicit could be


**If true, Kafka consumer offsets will be stored only in ScyllaDB table. If false, Kafka consumer offsets will be stored only in kafka (connector will not write offset information into ScyllaDB).**